### PR TITLE
feat(content): A Dead Man's Path again

### PR DESCRIPTION
**Pull request by bene-dictator**
**Content (Missions)**

## Summary
This is a short two-part mission where you help a pair of siblings follow their father's last will and testament. You get a special reward at the end if you stick along with them.

Thanks to @RisingLeaf for writing these missions. Since they left the main repository, I have taken up #7803 and updated it with the reviews from that PR.

~~Prime is now a French settlement. I needed some names for the planet's regions.~~

## Save File
This save file can be used to play through the new mission content:
[Tes Ting~Heartland, Cor Caroli.txt](https://github.com/endless-sky/endless-sky/files/11194990/Tes.Ting.Heartland.Cor.Caroli.txt)
I used Omnis to get the captain, if it matters.

## Artwork Checklist
The original PR had artwork made by RisingLeaf for these missions. For now I will leave out the artwork, but I will talk with RisingLeaf about it and see if we can sort something out, maybe in another PR.

## To-do
- [ ] Add a part about the Sheragi artefacts.
- [ ] Art???

### CODE REVIEW

File: data/human/human missions.txt
Comment (LazerLit) [inline data/human/human missions.txt:5755]:
```suggestion
			`	The only exception to the bustle and activity are a young woman and man, both dressed very formally, who are seemingly lost and having no luck approaching anybody. You watch them drift from one person to another, each time being sent off without an answer. Finally, they come towards you.`
```

File: data/human/human missions.txt
Comment (LazerLit) [inline data/human/human missions.txt:5761]:
```suggestion
			`	Her face brightens up with a smile, and she continues, "That's good to hear! My brother and I were searching for someone who could give us a ride to <destination>. We received notice that our father's last will is going to be published there. Do you think you could take us there? We will pay you for your service, of course."`
```

File: data/human/human missions.txt
Comment (LazerLit) [inline data/human/human missions.txt:5764]:
```suggestion
			`	She laughs. "There's no need to be suspicious, my brother and I were just searching for someone who could give us a ride to New China. We received notice that our father's last will is going to be published there. Do you think you could take us there? We will pay you for your service, of course.`
```

File: data/human/human missions.txt
Comment (LazerLit) [inline data/human/human missions.txt:5789]:
```suggestion
		dialog `When you arrive at <destination>'s spaceport, the siblings prepare to head off to the court where their father's last will will be read out. Just before they leave, Liam turns to you. "Captain <last>, I've transferred <payment> to your account. Would it be okay for you to wait here until we come back? We will need a ride home."`
```

File: data/human/human missions.txt
Comment (LazerLit) [inline data/human/human missions.txt:5836]:
```suggestion
		dialog `It is often thought that there are nothing interesting in these uninhabited systems by the Deep, but Sophie points out that there is a strange anomaly in this system. "When we got to see him, our father used to describe all these unusual sights he saw on his adventures flying through wormholes or nebulae. That anomaly reminds me of one of them."`
```

File: data/human/human missions.txt
Comment (LazerLit) [inline data/human/human missions.txt:5872]:
```suggestion
			`	After a short shuttlecraft flight on Prime's extensive and well-maintained shuttle service, you and the siblings arrive at a provincial city, where you can see a small mountain that stands alone against the rest of the landscape.`
```

File: data/human/human missions.txt
Comment (LazerLit) [inline data/human/human missions.txt:5875]:
```suggestion
			`	You take a maglev from the spaceport and get a tour through some of the regions of Prime's mainland; on the way you pass through the fields of New Toulouse, where the majority of the local agricultural industry is located, and into the New Pyrenees hills. After an hour you and the siblings step off at a provincial city. Although Betelgeuse starts to recede behind the horizon, you can see a small mountain that stands alone against the rest of the landscape.`
```

File: data/human/human missions.txt
Comment (LazerLit) [reply data/human/human missions.txt:5910]: I don't understand why they think the sword is different from the other artifacts. Perhaps some kind of explanation is needed?

**Review by LazerLit — CHANGES_REQUESTED** [review]

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:5789]:
```suggestion
		dialog `When you arrive at <planet>'s spaceport, the siblings prepare to head off to the court where their father's last will will be read out. Just before they leave, Liam turns to you. "Captain <last>, I've transferred <payment> to your account. Would it be okay for you to wait here until we come back? We will need a ride home."`
```
`<planet>` is better than `<destination>` in this instance.

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:5882]: The hissing noise is weak, although I could change it to something else.

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:5910]: I will think of a way to improve this paragraph.

**Comment by kestrel1110** [comment]
This looks amazing- would you be interested in getting a thumbnail for the sword, or do you intend to keep it as unknown?

**Comment by roadrunner56** [comment]
> This looks amazing- would you be interested in getting a thumbnail for the sword, or do you intend to keep it as unknown?

There was a thumbnail in the original PR, not sure why it's not included in this one.

**Comment by roadrunner56** [comment]
@bene-dictator Any specific reason the art files aren't in this PR? I thought they looked good :)

**Comment by bene-dictator** [comment]
I think RisingLeaf had some updates to the images that he did not commit yet to the PR? I still need to catch up with this, I have been busier lately.

File: data/human/human missions.txt
Comment (roadrunner56) [inline data/human/human missions.txt:5928]:
```suggestion
			`	You take the strange sword from Sophie. It's abnormally light, and you don't really know how to use it yet, but it has some strange feeling of power that seems to emanate from it.`
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]
One last thing. Other than the images this looks good to go.

**Review by roadrunner56 — APPROVED** [review]
Writing looks good, just need the art and this is good to go.

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt]: Okay, in the original PR there was a picture of the sword inserted in its pedestal, quite prominent over everything else. I think the reason I rewrote it this way was because the siblings believed that:
a. since their father had this sword by their side during his explorations, it should continued to be used or else "it would be wasted" on them
b. they couldn't take and study everything, given their limited resources

The thing about `a` is that the dialogue doesn't actually mention if the siblings know this, just that they say the sword has been. It is a narrative gap that I haven't thought about yet. A better explanation might be needed, but I will reread the whole thing and see if it really is needed as you've been the first to point this out.

Comment (roadrunner56) [reply data/human/human missions.txt]: I thought that they give you the sword because they don't get into situations where they could ever plausibly use it. If they keep it, it's kind of wasted on them since it'll never get used. This is why they give it to the player, who actually does get into fights and other situations where it could be useful.

Comment (LazerLit) [reply data/human/human missions.txt]: Ah, I see! I suppose I just thought there would be more to what the sword does than just chopping things. :-)

Plus, wouldn't they want to put an ancient artifact in a museum rather than have it destroyed through use? But I'm obviously just overthinking things.

Comment (bene-dictator) [reply data/human/human missions.txt]: > Ah, I see! I suppose I just thought there would be more to what the sword does than just chopping things. :-)

In fact, I did suggest more as well; I had an idea that whenever the sword was swung, an energy bolt like out of an Energy Blaster would be emitted. However, describing that in the dialogue would be clunky, and I don't think it's worth some dialogue and an event to update the sword.

> Plus, wouldn't they want to put an ancient artifact in a museum rather than have it destroyed through use? But I'm obviously just overthinking things.

Emerald Sword.

File: data/human/human missions.txt
Comment (tibetiroka) [inline data/human/human missions.txt:5764]:
```suggestion
			`	She laughs. "There's no need to be suspicious, my brother and I were just searching for someone who could give us a ride to <destination>. We received notice that our father's last will is going to be published there. Do you think you could take us there? We will pay you for your service, of course.`
```
The other one has this.

File: data/human/human missions.txt
Comment (tibetiroka) [inline data/human/human missions.txt:5770]: > with both heads looking down

Not a huge fan of this.

File: data/human/human missions.txt
Comment (tibetiroka) [inline data/human/human missions.txt:5814]: I don't think this explains those broad smiles.

File: data/human/human missions.txt
Comment (tibetiroka) [inline data/human/human missions.txt:5958]: > with their heads up

Not a huge fan of this one either.

File: data/human/outfits.txt
Comment (tibetiroka) [inline data/human/outfits.txt:406]: The middle ages ended about 1500 years before this, but swords have been in use after that as well.

**Review by tibetiroka — CHANGES_REQUESTED** [review]

File: data/human/outfits.txt
Comment (bene-dictator) [reply data/human/outfits.txt:406]: Yes, but the description shows the contemporary view on swords and exaggerating "thousands of years ago" reflects that they're just relics of an old barbaric age. I am happy to change it but I don't think it would look better.

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:5770]: Why not? The next best option I can think of is:
```suggestion
			`	Her smile fades away as you say that, but they both simply nod and walk away out the gates of the spaceport, their heads down.`
```
but it seems like you'd like to avoid that sort of expression?

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:5814]: I wrote it under the impression that they would be very excited at the chance of a trip around the entirety of human space in addition to fulfiling their father's wishes and being able to experience his life in some way. After all they expected to be sadder and now they have some sort of mystery on their hands.

File: data/human/human missions.txt
Comment (LorenzoBolla) [inline data/human/human missions.txt:5998]: Once again, i like commas lol

Also, god damn! This whole tablet part gave me _chills_. I love it!
```suggestion
			`			"This room you are in, now, is the culmination of all my work. I hope you understand that it was for your sake that I kept these a secret. My journey was long and unforgiving, but it was my life. I hope you continue that journey, to understand the universe a little better.`
```

File: data/human/outfits.txt
Comment (LorenzoBolla) [reply data/human/outfits.txt:406]: Seems fine by me, bene!

**Review by LorenzoBolla — CHANGES_REQUESTED** [review]
Awesome work was put into this mission! I love it. Here's my suggestions:

File: data/human/human missions.txt
Comment (tibetiroka) [reply data/human/human missions.txt:5861]: Yeah. It's probably fine though.

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:5954]: Hmm, this doesn't flow as well to me. Not wrong to me really, but not as nice sounding.

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:5998]: Yeah this doesn't flow as well for me too.

File: data/human/outfits.txt
Comment (Saugia) [inline data/human/outfits.txt:406]: "Received this from..." sounds very odd as a starting line for an outfit description. That can be adjusted.

 I think the whole first chunk here can be omitted as you know where you got the sword from, the whole adventure is supposed to leave its mark on the player rather than telling them as an afterthought. You also know this through the logs.

**Review by Saugia — CHANGES_REQUESTED** [review]
Some minor things. While pending artwork, I had thought it would almost be better not to have artwork so that it's up to the player's imagination where the sword came from and what race. If it is specifically a Sheragi weapon, I'd ask for a model based on that, preferably from @Karirawri. Although there was a whole room scene in the original PR, I think only an outfit artwork is necessary so the player can imagine the visuals of a whole large alien artifact containing dome, as opposed to the lone sword in the original.

**Review by Saugia — CHANGES_REQUESTED** [review]
Some minor things. While pending artwork, I had thought it would almost be better not to have artwork so that it's up to the player's imagination where the sword came from and what race. If it is specifically a Sheragi weapon, I'd ask for a model based on that, preferably from @Karirawri. Although there was a whole room scene in the original PR, I think only an outfit artwork is necessary so the player can imagine the visuals of a whole large alien artifact containing dome, as opposed to the lone sword in the original.

**Comment by Saugia** [comment]
Bruh, Github.

**Review by W1zrad — CHANGES_REQUESTED** [review]
Lookin' good! Just some writing edits. I guess the only other thing we're waiting on is the sword art, right? It was suggested on-stream that maybe the sword doesn't strictly need artwork (maybe it'd be fun to leave it up to the reader's imagination), but I'm still leaning towards including art. See @Saugia 's comment for more.

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:5927]: I'll make the first suggestion, but those systems where Terminus is are really by the region of the Deep, aren't they?

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:5929]: How about this?
```suggestion
		dialog `While Sophie and Liam are both quite chatty at times, the beauty of this star silences them with its awe.`
```

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:5927]:
```suggestion
		dialog `It is often thought that there's nothing interesting in these uninhabited systems by the Deep, but Sophie points out that there is a strange anomaly in this system. "When we got to see him, our father used to describe all these unusual sights he saw on his adventures flying through wormholes or nebulae. That anomaly reminds me of one of them."`
```

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:5941]: "finally arriving at their final destination" feels tautological.

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:5988]: Yeah, I think it was a little bit of an afterthought in the old PR. I can write a little more, but I don't know enough about the Sheragi to say too much about it (and I'm busy at the moment) so it might take some time.

File: data/human/outfits.txt
Comment (bene-dictator) [inline data/human/outfits.txt:405]:
```suggestion
```

File: data/human/human missions.txt
Comment (bene-dictator) [reply data/human/human missions.txt:5873]: I prefer it without the split, it makes him sound more excited to me.

File: data/human/human missions.txt
Comment (W1zrad) [reply data/human/human missions.txt:5942]: Oops, good catch! Meant this:
```suggestion
			`The siblings are uncharacteristically quiet, perhaps in anticipation of arriving at their destination. It is only when you have all stepped out into Prime's spaceport that Sophie says, "Captain <last>, you've helped us immensely, and we can't thank you enough for bringing us all across the galaxy! But we have one last request. Both of us think that you should come with us to see what our father left behind. Do you want to see it?"`
```

File: data/human/outfits.txt
Comment (W1zrad) [inline data/human/outfits.txt:405]:
```suggestion
	description "While swords are seen today as being old curios and commemorative objects, rendered useless when gunpowder became dominant on the battlefield thousands of years ago, this sword is unlike any you have seen before. Despite how heavy it looks, it is remarkably light, and there are inscriptions on the blade itself that don't appear to be like any human language."
```

**Comment by W1zrad** [comment]
Going to add the waiting label until the art situation is figured out.

**Comment by bene-dictator** [comment]
Update: it's been mentioned that a Sheragi sword would be too big for a human to handle. What I've come up with, but not got down in writing yet, is that the sword would be a Hai sword. There were two ways I thought this could go; the sword was a gift from the Hai from initial human (not Alpha) contact, or the sword was stolen by the father.

Right now I am planning the second option. Maybe the father was less honest to his children about describing himself as an adventurer, and that his secret stash was not entirely clean which is why he needs to hide it like he has : )

Although I have exams in a few weeks, I'm thinking of getting back to this in my spare time afterwards.

**Comment by bene-dictator** [comment]
I will get back to this after my exams. Had writer's block these holidays. One day I'll get this one done, RisingLeaf

**Comment by TheGiraffe3** [comment]
Would the artwork in the original PR satisfy that constraint?
As in: do the art reviewers think that the original PR's artwork would be good enough if the writing side of this was handled?

**Comment by TheGiraffe3** [comment]
@Saugia do you have thoughts on the above?

**Comment by Saugia** [comment]
https://github.com/endless-sky/endless-sky/pull/8654#pullrequestreview-1427119824 was my last comment on this regarding art. If you're considering re-opening this PR, I would do so without artwork in mind for now, since the original artist isn't around at the moment to make any changes. We can always add art later- it doesn't have to be the reason you don't PR this.

**Comment by TheGiraffe3** [comment]
I think the latest review comment on the art was addressed; the pull request seems to have been waiting on your approval art-wise.
I do intend to re-PR this if bene is alright with it, but the artwork would have been an added bonus.

**Comment by bene-dictator** [comment]
There were additional things I wanted to do with the mission. But I am not sure of how they would fit. It involved changing the Sheragi sword to a Hai one, for example (as Kari correctly pointed out that a Sheragi sword large enough for a human to wield would probably be like a toothpick to the Sheragi). But Hurl has also stated that circumstances would have to be significant enough for the Hai to give a human a Hai sword. (Though, I'm sure something can be done to justify this.) I also wanted to make it a bit more interactive, rather than just a tour of the galaxy, but I ran out of time and bother before that. I could still work on it in a couple weeks – I don't want to break that promise – but I'm overseas for a couple months so... ~~no promises?~~

**Pull request closed without merging**
